src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Module/Startup.cs

```diff
@@ -12,6 +13,7 @@
 using OrchardCore.Templates.Cms.Module.Handlers;
 using OrchardCore.Templates.Cms.Module.Models;
 using OrchardCore.Templates.Cms.Module.Settings;
+using OrchardCore.Templates.Cms.Module.ViewModels;
```

Comment:
Think you might need a `using OrchardCore.Liquid` to get `TemplateOptions`

Reply:
TemplateOptions is in Fluid.